src/accel/cuda/GPUSolver.cu

```diff
@@ -1,22 +1,19 @@
 #include "GPUSolver.h"
 
-/** The number of azimuthal angles */
-__constant__ int num_azim[1];
+/** The number of FSRs */
+__constant__ long num_FSRs;
```

Comment:
nope, that's just the way I found it. I agree they should be in the header.

Reply:
Actually, it seems that for CUDA, you have to put __constant__ variables in the implementation file and not the header, otherwise you get multiple definition errors.

Reply:
Ok good to know. Thanks for checking!